crypto/fipsmodule/rand/urandom_test.cc

```diff
@@ -420,13 +419,14 @@ static std::vector<Event> TestFunctionPRNGModel(unsigned flags) {
     };
   }
 
-  const size_t kSeedLength = CTR_DRBG_ENTROPY_LEN * (is_fips ? 10 : 1);
+  const size_t kSeedLength = CTR_DRBG_ENTROPY_LEN;
   const size_t kAdditionalDataLength = 32;
 
   if (!have_rdrand()) {
     if ((!have_fork_detection() && !sysrand(true, kAdditionalDataLength)) ||
-        // Initialise CRNGT.
-        (is_fips && !sysrand(true, 16)) ||
+        // Initialise CRNGT. In FIPS mode we use a blocking sysrand call for
```

Comment:
I may need to look at this again in `rand.c`, but I see that, though Jitter changes affect `rand_get_seed()`, there is a still a difference in some calls between being in FIPS build or not, so why `is_fips` checks are removed from the conditions of this if-statement?

Reply:
The sequence of calls in FIPS and non-FIPS is the same. The difference is in the purpose of the calls to `sysrand`, in FIPS mode we call it to get personalization string for DRBG init or additional data for reseed, while in non-FIPS mode we call it to get the entropy. I tried to capture this with the comment, let me know if it needs more explanation.

Reply:
I suggest you add the following to the comment: Note that the length of the personalization string is the same as that of the seed.

Reply:
done